lms/lmsweb/views.py

```diff
@@ -199,7 +199,9 @@ def _create_comment(
     )
 
     return jsonify({
-        'success': 'true', 'id': comment_.id, 'text': comment_.comment.text,
+        'success': 'true', 'text': comment_.comment.text,
```

Comment:
instead of re-type the hard codded keys here, consider formatting using the same method